src/components/Main/Header/Header.tsx

```diff
@@ -9,11 +10,44 @@ interface Props {
 
 function Header(props: Props) {
   return (
-    <div id={props.id}>
-      <h2>HELLO WORLD</h2>
-      <h1>WE'RE MAKING WAVES</h1>
-      <h2>GWC at UVA</h2>
-      <img src="../../assets/images/main-header-wave.svg" alt="Orange wave" />
+    <div id={props.id}> 
+
+    <div className="container-fluid m-0 bg-orange">
+      <Row lg={2}>
+      <Col lg={1}><h4></h4></Col>
+      </Row>
+      <Row lg={3}>
+        <Col lg={1}></Col>
+        <Col lg={10}><h2 className="text-light-grey">HELLO WORLD</h2></Col>
```

Comment:
color should be peach -- figma for reference https://www.figma.com/file/3oCYmtWALN20gRkSa5Ew7V/GWC-General-Site?node-id=460%3A14

mobile version is on the figma as well